lua/entities/gmod_wire_expression2/core/hologram.lua

```diff
@@ -1227,6 +1227,10 @@ e2function void holoParent(index, entity ent)
 	local Holo = CheckIndex(self, index)
 	if not Holo then return end
 
+	if not E2Lib.isOwner(self, ent) then
+		return self:throw("You do not have permission to parent to this entity", nil)
+	end
+
```

Comment:
Would suggest placing this before the `CheckIndex`call so that isn't called needlessly.